.github/workflows/remove-long-timeout-labels.yml

```diff
@@ -76,21 +77,20 @@ jobs:
 
       - name: Check if CI was restarted
         id: check
+        shell: brew ruby {0}
```

Comment:
Not for now but IMO ideally all `brew ruby` usage like this would be moved into a Homebrew/brew command.